ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -141,7 +141,8 @@ def prepare_reverse_zone(host, ip):
     zone = get_reverse_zone_default(ip)
     result = host.run_command(["ipa",
                       "dnszone-add",
-                      zone], raiseonerr=False)
+                      zone,
+                      "--skip-overlap-check"], raiseonerr=False) # To force dnszone-add
```

Comment:
The place of the comment is confusing. It's unclear if the comment refers to ``raiseonerror=False`` or ``--skip-overlap-check``.

Please reformat the code as
```
    result = host.run_command(
        [
            "ipa",
            "dnszone-add",
            "--skip-overlap-check",  # to force dnszone-add
            zone
        ],
        raiseonerr=False
    )
```
or perhaps replace the call with ``add_dns_zone(host, zone, skip_overlap_check=True)``.